src/hooks/useBooksApi.js

```diff
@@ -6,7 +6,7 @@ import { loadAuthToken } from '../utils/local-storage';
 export default function useGetBooks(booksItemCount, pageNumber, sortBy) {
 
 
-  const getAllBooks = api.get(`/books`,{booksItemCount, pageNumber, sortBy}) 
+  const getAllBooks = !localStorage.authToken ? {data: []} : api.get(`/books`,{booksItemCount, pageNumber, sortBy}) 
```

Comment:
This definitely improves user experience, it's better than showing the "error fetching books" error after they log in. However, it's still not ideal because the user does not see their books on initial load. I have a solution for this in my PR you can check out, I will go over it next meetup.